overlay/sbin/k3os-upgrade-kernel

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ $(whoami) != "root" ]
+then
+	echo "This script must be run as root."
+	exit 1
+fi
+
+PROC=$(uname -m)
+
+# Take a parameter of the version number (i.e. v0.4.0) if it is given, otherwise use latest
+if [ -z $K3OS_VERSION ]
+then
+	K3OS_VERSION=$(curl -sL api.github.com/repos/rancher/k3os/releases/latest | jq .tag_name -r)
```

Comment:
We should be able to use `latest` as the default version, we just need to update the download links. The following link work. https://github.com/rancher/k3os/releases/latest/download/k3os-kernel-version-amd64.
```suggestion
	K3OS_VERSION=latest
```